portal-web/test/functional/com/liferay/portalweb/tests/enduser/headless/HeadlessDeliveryAPI/HeadlessDeliveryBatchMode.testcase

```diff
@@ -24,18 +59,16 @@ definition {
 				headline = "headline of my blog posting");
 		}
 
-		task ("And Given I use POST headless-delivery/v1.0/blog-postings/{blogPostingId}/comments/batch?createStrategy=INSERT to create two base comments") {
+		task ("When I use POST headless-delivery/v1.0/blog-postings/{blogPostingId}/comments/batch?createStrategy=INSERT to create two base comments") {
```

Comment:
I have used the opportunity of filling the missing "Given,When,Then" syntax to reorganize a bit the steps so it's more declarative.